lib/std/os.zig

```diff
@@ -2802,6 +2802,8 @@ pub const AcceptError = error{
     /// by the socket buffer limits, not by the system memory.
     SystemResources,
 
+    Invalid,
```

Comment:
```suggestion
    SocketNotListening,
```

Can you also put in the doc comments for this error the explanation you put in the PR about shutdown?